libs/src/services/creatorNode/CreatorNodeSelection.test.js

```diff
@@ -227,9 +235,9 @@ describe('test CreatorNodeSelection', () => {
         longitude: '-122.4619'
       } })
 
-    // cold, overnight pizza -- behind by minor version, fast. you kind of want this
-    const shouldBeSecondary1 = 'https://shouldBeSecondary1.audius.co'
-    nock(shouldBeSecondary1)
+    // cold, overnight pizza -- behind by minor version, fast. nope
```

Comment:
LOL. i love that you've now changed your stance on cold, overnight pizza
(i agree with your former take, gotta eat it 🍕)